backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -48,9 +49,11 @@ public ResponseEntity<String> resendUserEmail(@RequestParam String email) {
     return ResponseEntity.ok(response);
   }
 
-  @GetMapping
-  public String getUser() {
-    //  TODO
-    return "test";
+  @GetMapping()
+  public List<UserResponse> getUsers() {
+    List<UserDto> listOfUsers = userService.listAllUsers();
+    return listOfUsers.stream()
+        .map(e -> modelMapper.map(e, UserResponse.class))
```

Comment:
In userService you are mapping it in UserDto, and then you are mapping to UserResponse, why are you using two model objects instead of one ?

Reply:
Yeah, you can do that in another PR, and to fix that for all other endpoints in UserController and to exclude model mapper from this class. You can create a ticket for that to do after this one.

Reply:
Sure, we are due for refactoring anyway :)